rust/agama-cli/src/server.rs

```diff
@@ -0,0 +1,195 @@
+use clap::Subcommand;
+use reqwest::header::{HeaderMap, HeaderValue, CONTENT_TYPE};
+use std::fs::File;
+use std::io;
+use std::io::{BufRead, BufReader};
+use std::path::PathBuf;
+
+const DEFAULT_JWT_FILE: &str = "/tmp/agama-jwt";
```

Comment:
I would not save the token in `/tmp`. I would rather prefer to save it in some private place, like `$HOME/.agama/...`.